app/api/domains/api.py

```diff
@@ -109,6 +116,77 @@ def format_map_basic(m: Beatmap) -> dict[str, object]:
     }
 
 
+@router.get("/calculate_pp")
+async def api_calculate_pp(
+    token: HTTPCredentials = Depends(oauth2_scheme),
+    beatmap_id: int = Query(None, alias="id", min=0, max=2_147_483_647),
+    n100: int = Query(0, max=2_147_483_647),
+    n50: int = Query(0, max=2_147_483_647),
+    misses: int = Query(0, max=2_147_483_647),
+    mods: int = Query(0, min=0, max=2_147_483_647),
+    mode: int = Query(0, min=0, max=11),
+    combo: int = Query(None, max=2_147_483_647),
+    acclist: Optional[str] = Query(None, alias="acc"),
```

Comment:
hey, better to use industry standard here for passing multiple query parameters,
`?accuracy=12.34&accuracy=32.65&accuracy=43.76`

this can be accomplished by using `list[str]` as the type, fastapi will handle the magic for you

Reply:
actually, you can probably use `list[float]`

Reply:
oh alright